### PR TITLE
Chapter 7 of Tutorial

### PR DESCRIPTION
**Pull request by prhomhyse**
This is the complete chapter 7 up for review.

### CODE REVIEW

File: part-07.mdx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+ Click on the top download icon as seen on the short clip below. 
+
+![import-shot](https://user-images.githubusercontent.com/12263278/64901503-ee696f80-d691-11e9-9678-8d0e2a162402.gif)
+
+ Select the Netdata snapshot file to import. 
+
+ Once the file is loaded, click `import` to render it.
+
+ ## Up Next
```
Comment (joelhans) [inline part-07.mdx]: Change this to **What's next?**

```suggestion
 ## What's next?
```

File: part-07.mdx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+## Check if there's an update to Netdata
+
+Netdata upon installation with [the one-liner](https://docs.netdata.cloud/packaging/installer/#one-line-installation) 
+comes with the auto-update option enabled by default.
+
+You can always check if there is an update available right from the Update tab of your Netdata dashboard.
+
+![nedata-update](https://user-images.githubusercontent.com/12263278/64876743-be957a00-d647-11e9-83dd-2f0a8df572cb.gif)
+
+You can see an update is available from the above short screen record.
```
Comment (joelhans) [inline part-07.mdx]:
```suggestion
If an update is available, you'll see a modal similar to the one above.
```

File: part-07.mdx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+Netdata automatically updates itself, but if you choose to update it manually, there are [several well-documented methods](https://docs.netdata.cloud/packaging/installer/update/) to achieve that. 
+However, it is best practice for you to first go over the [changelog](https://docs.netdata.cloud/changelog/).
+
+## Export or import a snapshot
+
+Snapshot files include all the information of the dashboard, including the URL of the origin server, 
+its netdata unique ID, and chart data queries for the visible timeframe,
```
Comment (joelhans) [inline part-07.mdx]:
```suggestion
its unique ID, and chart data queries for the visible timeframe,
```

File: part-07.mdx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+so align, pan and zoom the charts as needed.
```
Comment (joelhans) [inline part-07.mdx]: I think you meant "to"?

```suggestion
to align, pan, and zoom the charts as needed. 
```

File: part-07.mdx [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+However, it is best practice for you to first go over the [changelog](https://docs.netdata.cloud/changelog/).
+
+## Export or import a snapshot
+
+Snapshot files include all the information of the dashboard, including the URL of the origin server, 
+its netdata unique ID, and chart data queries for the visible timeframe,
+so align, pan and zoom the charts as needed. 
+Snapshots are not in real-time. 
+
+Netdata can export and import dashboard snapshots. Any Netdata can import the snapshot of any other Netdata.
```
Comment (joelhans) [inline part-07.mdx]: I feel like this should be the first sentence of the section, then follow up with what a snapshot contains.

File: part-07.mdx [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+ finally select your desired compression method and click `Export`.
+
+ ![export-shot](https://user-images.githubusercontent.com/12263278/64901454-48b60080-d691-11e9-9c14-1539bc841735.gif)
+
+ You save the snapshot file after it gets generated on your browser and like a regular file attachment, 
+ you can email it to your colleague.
+
+ Congratulations, you have successfully exported a snapshot.
+
+ #### To import or load a snapshot
```
Comment (joelhans) [inline part-07.mdx]: Shouldn't this be `###`?

```suggestion
 ### To import or load a snapshot
```

File: part-07.mdx [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+## Export or import a snapshot
+
+Snapshot files include all the information of the dashboard, including the URL of the origin server, 
+its netdata unique ID, and chart data queries for the visible timeframe,
+so align, pan and zoom the charts as needed. 
+Snapshots are not in real-time. 
+
+Netdata can export and import dashboard snapshots. Any Netdata can import the snapshot of any other Netdata. 
+
+You download the snapshot to your computer as a file,
```
Comment (joelhans) [inline part-07.mdx]: I'm not sure if the content from here to **To export or save a snapshot** is necessary.

File: part-07.mdx [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+ #### To import or load a snapshot
+
+ Click on the top download icon as seen on the short clip below. 
+
+![import-shot](https://user-images.githubusercontent.com/12263278/64901503-ee696f80-d691-11e9-9678-8d0e2a162402.gif)
+
+ Select the Netdata snapshot file to import. 
+
+ Once the file is loaded, click `import` to render it.
```
Comment (joelhans) [inline part-07.mdx]: This is the point where I'd explain that refreshing Netdata will then erase the snapshot data and bring you back to real-time updates.

File: part-07.mdx [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+
+Netdata can export and import dashboard snapshots. Any Netdata can import the snapshot of any other Netdata. 
+
+You download the snapshot to your computer as a file, 
+that can be imported back into any Netdata dashboard, 
+you do not need to import it back to the current server.
+
+Snapshots don't upload to a server; instead, the web browser on your computer handles them.
+
+They are temporary because, on the reload of your browser, the actual Netdata dashboard replaces the snapshot.
```
Comment (joelhans) [inline part-07.mdx]: Cut this, since I think you should put it later.

File: part-07.mdx [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+
+-   Change the dashboard's settings
+-   Check if there's an update to Netdata
+-   Export or import a snapshot
+
+Let's get started!
+
+## Change the dashboard's settings
+
+The settings tab at the top of your Netdata dashboard houses browser settings.
```
[the remark this answers is not in the document]
Comment (prhomhyse) [reply part-07.mdx]: I kind of prefer area.

**Pull request merged**